src/python/grpcio/grpc/_channel.py

```diff
@@ -1533,22 +1551,25 @@ class _StreamStreamMultiCallable(grpc.StreamStreamMultiCallable):
     _request_serializer: Optional[SerializingFunction]
     _response_deserializer: Optional[DeserializingFunction]
     _context: Any
+    _registered_method: bool
 
     # pylint: disable=too-many-arguments
     def __init__(
         self,
         channel: cygrpc.Channel,
         managed_call: IntegratedCallFactory,
         method: bytes,
-        request_serializer: Optional[SerializingFunction] = None,
```

Comment:
Nice catch!

Note to self: This is not a backward-incompatible change because the _constructor_ of this class is a private API. It's just the other methods that constitute public API because instances of this class are returned from `Channel.stream_stream`